modules/virt-enabling-usb-host-passthrough.adoc

```diff
@@ -0,0 +1,60 @@
+// Module included in the following assemblies:
+//
+// * virt/advanced_vm_management/virt-configuring-usb-host-passthrough.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="virt-enabling-usb-host-passthrough_{context}"]
+= Enabling USB host passthrough
+
+You can enable USB host passthrough at the cluster level.
+
+You specify a resource name and USB device name for each device you want to add and then assign to a virtual machine (VM). You can allocate more than one device, each of which is known as a `selector` in the HyperConverged (HCO) custom resource (CR), to a single resource name. If you have multiple, identical USB devices on the cluster, you can choose to allocate a VM to a specific device.
+
+.Prerequisites
+
+* Enabling the passthrough requires cluster administrator privileges.
+
+.Procedure
+
+. Identify the USB device vendor and product by running the following command:
++
+[source,terminal]
+----
+$ lsusb
+----
+
+. Open the HCO CR by running the following commmand:
++
+[source,terminal]
+----
+$ oc edit hyperconverged kubevirt-hyperconverged -n openshift-cnv
+----
+
+. Add a USB device to the `permittedHostDevices` stanza, as shown in the following example:
+
++
+.Example YAML snippet
+[source,yaml]
+----
+apiVersion: hco.kubevirt.io/v1beta1
+kind: HyperConverged
+metadata:
+   name: kubevirt-hyperconverged
+   namespace: {CNVNamespace}
+spec:
+  configuration:
+    permittedHostDevices: <1>
+      usb: <2>
```

Comment:
it's named `usbHostDevices`, see:
https://github.com/kubevirt/hyperconverged-cluster-operator/blob/53e7b476314f476313966f7f8f181ce48552223d/api/v1beta1/hyperconverged_types.go#L525

Reply:
@tiraboschi, I'm not sure what should be updated. Should **permittedHostDevices** become **PermittedHostDevices**? And should **usb** become **USBHostDevices**?

Reply:
```yaml
---
apiVersion: hco.kubevirt.io/v1beta1
kind: HyperConverged
metadata:
  name: kubevirt-hyperconverged
  namespace: {CNVNamespace}
spec:
  permittedHostDevices:
    usbHostDevices:
    - resourceName: kubevirt.io/peripherals
      selectors:
        - vendor: "045e"
          product: "07a5"
        - vendor: "062a"
          product: "4102"
        - vendor: "072f"
          product: "b100"
```